examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -41,6 +41,16 @@
 
 #include "ble_config.h"
 
+#if SL_ICD_ENABLED && SIWX_917
```

Comment:
These changes were already added as part of my PR. Rebase the PR these should be removed

Reply:
rebased

Reply:
rebased the brach